app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -109,6 +102,79 @@ type MonitoringLocationGroups = {
   },
 };
 
+function useStreamgageData(fetchedData, fetchStatus) {
```

Comment:
Can you move this custom hook to the `src/utils/hooks.js` file and just import it into the Monitoring, Overview, and LocationMap components?

Reply:
I had thought about moving it there, but I hesitated because @courtneymyers said he's not a fan of populating that hooks file with too much. But I do think it makes sense in this case, since it's used in three places.

Reply:
Thanks, @maxdiebold-erg – my concern was more around exporting functions that are only imported elsewhere _once_, and you're right I'd like to see that `/utils/hooks` file slimmed down. But if you're creating a hook that's used across multiple files, it'd make sense to move it there as @cschwinderg requested (at least  with our current setup – that may change at some point, depending on if we restructure the files).

Reply:
Cool, just figured I'd check! Thanks.